ldms/src/ldmsd/ldmsd.c

```diff
@@ -436,6 +437,16 @@ const char *ldmsd_loglevel_to_str(enum ldmsd_loglevel level)
 	return "LDMSD_LNONE";
 }
 
+void ldmsd_inc_cfg_cntr()
+{
+	__sync_fetch_and_add(&cfg_cntr, 1);
```

Comment:
From https://gcc.gnu.org/onlinedocs/gcc/_005f_005fsync-Builtins.html:

> These functions are implemented in terms of the ‘__atomic’ builtins (see __atomic Builtins). They should not be used for new code which should use the ‘__atomic’ builtins instead.

So, I suggest that you use `__atomic_fetch_add(&cfg_cntr, 1, __ATOMIC_SEQ_CST)` instead.

Reply:
Thanks @narategithub, I'll update that. I actually ran into the __atomic builtins when I was implementing this, but figured I'd try to stay consistent with the existing code.